elections/steering/2024/README.md

```diff
@@ -224,17 +233,9 @@ roles you may hold.
 
 ## Voting Process
 
-Contributors may check their voter eligibility at any time once the election
-process starts, by going to the [election app], logging in, navigating to
-the current year election, and seeing if the screen there says that they are eligible.
-That screen takes its data from the [voters.yaml] file.
-
-If the app does not say that you are eligible, because you have worked on
-Kubernetes in a way that is NOT reflected in GitHub contributions, you can use
-the exception form built into the same app to ask to participate in the election.
-
 Elections will be held using time-limited [Condorcet] ranking on [Elekto].
 The most preferred candidates will be elected to the open seats.
+To get more info about voting eligibility criteria, refer to the [eligibility section](#eligibility). 
```

Comment:
Please move this up to the beginning of the section, and better text:

```See the [eligibility section](#eligibility) to understand if you are eligible to vote.```

Reply:
Great, done!